internal/install/discovery/psutil_discoverer.go

```diff
@@ -67,3 +70,41 @@ func (p *PSUtilDiscoverer) Discover(ctx context.Context) (*types.DiscoveryManife
 
 	return &m, nil
 }
+
+func filterValues(m types.DiscoveryManifest) types.DiscoveryManifest {
+	if !isValidOpenInstallationPlatform(m.Platform) {
+		m.Platform = ""
+	}
+
+	if !isValidOpenInstallationPlatformFamily(m.PlatformFamily) {
+		m.PlatformFamily = ""
+	}
+
+	return m
+}
+
+func isValidOpenInstallationPlatform(platform string) bool {
+	s := reflect.ValueOf(&types.OpenInstallationPlatformTypes).Elem()
+
+	for i := 0; i < s.NumField(); i++ {
+		v := s.Field(i).Interface().(types.OpenInstallationPlatform)
+		if string(v) == platform {
+			return true
+		}
+	}
+
+	return false
+}
+
+func isValidOpenInstallationPlatformFamily(platformFamily string) bool {
+	s := reflect.ValueOf(&types.OpenInstallationPlatformFamilyTypes).Elem()
+
+	for i := 0; i < s.NumField(); i++ {
+		v := s.Field(i).Interface().(types.OpenInstallationPlatformFamily)
+		if string(v) == platformFamily {
```

Comment:
@ctrombley Is this a case insensitive comparison? I believe we might want that.

Reply:
I believe that graphQL is case-sensitive when checking enums.